packages/cosmic-swingset/src/chain-main.js

```diff
@@ -308,7 +310,8 @@ export default async function main(progname, args, { env, homedir, agcc }) {
     /** @type {CosmosSwingsetConfig} */
     const swingsetConfig = harden(initAction.resolvedConfig || {});
     validateSwingsetConfig(swingsetConfig);
-    const { slogfile } = swingsetConfig;
+    const { slogfile, vatTranscriptRetention } = swingsetConfig;
+    const keepTranscripts = vatTranscriptRetention === 'archival';
```

Comment:
In case we decide to support "replay" ?

```suggestion
    const keepTranscripts = vatTranscriptRetention !== 'operational';
```